src/main/kotlin/no/nav/syfo/melding/domain/MeldingTilBehandler.kt

```diff
@@ -111,14 +111,14 @@ fun MeldingTilBehandler.toJournalpostRequest(pdf: ByteArray) =
 
 fun MeldingTilBehandler.createTittel(): String {
     return when (this.type) {
-        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER -> MeldingTittel.DIALOGMELDING_DEFAULT.value
+        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER, MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING -> MeldingTittel.DIALOGMELDING_DEFAULT.value
         MeldingType.FORESPORSEL_PASIENT_PAMINNELSE -> MeldingTittel.DIALOGMELDING_PAMINNELSE.value
     }
 }
 
 fun MeldingTilBehandler.createOverstyrInnsynsregler(): String? {
     return when (this.type) {
-        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER -> OverstyrInnsynsregler.VISES_MASKINELT_GODKJENT.value
+        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER, MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING -> OverstyrInnsynsregler.VISES_MASKINELT_GODKJENT.value
```

Comment:
Må legges til her for at Forespørsel legeerklæring skal vises for bruker på mine saker.